app/src/main/java/io/github/droidkaigi/confsched2018/data/db/entity/mapper/SessionDataMapperExt.kt

```diff
@@ -47,6 +49,18 @@ fun SessionWithSpeakers.toSession(
     )
 }
 
+fun SessionFeedbackEntity.toSessionFeedback(): SessionFeedback = SessionFeedback(
```

Comment:
Mmm, I guess it's better to use inner join to grab session titles. 🤔 
Then, getAllSessionFeedback's sql should be like `SELECT session_feedback.*, session.title as session_title FROM session_feedback INNER JOIN session ON session.id = session_feedback.session_id` and SessionFeedbackEntity would need `session_title` field with Ignore annotation.

If no, https://github.com/DroidKaigi/conference-app-2018/pull/474/files#diff-279f886f71ad2a56e07db71bda11d857R88 should throw an error if no session is found.

Reply:
Entity and Dao support done.
Although there is still a problem that it is not taken if it is an application though the title can be obtained by executing the query on SQLite. so I create another issue 🙇